packages/atomic/src/components/facets/facet-search/facet-search.pcss

```diff
@@ -3,7 +3,7 @@
 }
 
 .search-results {
-  @apply z-10 absolute w-full bg-background border-divider-t-0 empty:border-none rounded-b overflow-y-scroll left-0;
+  @apply z-10 absolute w-full bg-background border-divider-t-0 empty:border-none rounded-b overflow-y-auto left-0;
```

Comment:
On mac, the scrollbar is not as visible by default OS settings, it only appears when you actually scroll.